cluster-sync/sync-os-ci.sh

```diff
@@ -9,7 +9,14 @@ source ./hack/build/config.sh
 source ./hack/build/common.sh
 source ./cluster-up/hack/common.sh
 source ./cluster-up/cluster/${KUBEVIRT_PROVIDER}/provider.sh
-source ./cluster-sync/${KUBEVIRT_PROVIDER}/provider.sh
+
```

Comment:
I think file can go as well, I don't think we ever managed to integrate with openshift ci.

Reply:
Cool!